src/main/java/com/fuem/controllers/FeedbackController.java

```diff
@@ -0,0 +1,76 @@
+package com.fuem.controllers;
+
+import com.fuem.models.Feedback;
+import com.fuem.repositories.FeedbackDAO;
+import com.fuem.models.User; // Đảm bảo bạn đã nhập User model
+import java.io.IOException;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.annotation.WebServlet;
+import jakarta.servlet.http.HttpServlet;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import jakarta.servlet.http.HttpSession;
+
+@WebServlet(name = "FeedbackController", urlPatterns = {"/FeedbackController"})
```

Comment:
urlPattern ngắn gọn thôi mô tả action đấy thôi cậu
```suggestion
@WebServlet(name = "FeedbackController", urlPatterns = {"/student/feedback"})
```